components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1430,12 +1432,48 @@ void RewardsServiceImpl::SetCatalogIssuers(const std::string& json) {
   bat_ledger_->SetCatalogIssuers(json);
 }
 
+std::vector<uint64_t> RewardsServiceImpl::GetEarningsRange() {
+  uint64_t difference;
+  auto base_time_now = base::Time::Now();
+  int one_day = (60 * 60 * 24);
+
+  base::Time::Exploded exploded;
+  base_time_now.LocalExplode(&exploded);
+
+  if (exploded.day_of_month > 5) {
+    difference = ((exploded.day_of_month - 5) * one_day);
+  } else {
+    difference = (exploded.day_of_month * one_day);
+  }
+
+  uint64_t now = static_cast<uint64_t>(std::time(nullptr));
```

Comment:
see comment above, please use base::Time and related methods

Reply:
👍 fixed